classification2.Rmd

```diff
@@ -170,6 +171,7 @@ set.seed(4235)
 random_numbers <- sample(0:9, 10, replace=TRUE)
 random_numbers
 
+set.seed(4235)
```

Comment:
I don't think this is correct -- why add this?

Reply:
This is the original version: 
<img width="516" alt="Screen Shot 2022-04-19 at 11 42 39 AM" src="https://user-images.githubusercontent.com/3838091/164073844-0726ef25-abfa-4c16-9c6a-7914962e2d6e.png">

So we are getting different numbers

Reply:
Or is that what you wanted to show? Perhaps I misunderstood the paragraph

Reply:
same comment as above